src/app/helpers/LanguageHelper.ts

```diff
@@ -27,17 +27,17 @@ export class LanguageHelper {
     return JSON.parse(fs.readFileSync(LanguageHelper.getLanguageFileLocation(language)).toString());
   }
 
-  public getTranslations(): any {
-    return LanguageHelper.getTranslations(
-      LanguageHelper.isSupported(this.preferredLanguage) ? this.preferredLanguage : "en"
-    );
+  public getPreferredLanguage(): string {
```

Comment:
this is exactly the same, right? no changes?

Reply:
No change